tests/xtro-sharpie/macOS-WebKit.todo

```diff
@@ -1,10 +1,2 @@
-!missing-field! WKWebsiteDataTypeFetchCache not bound
-!missing-field! WKWebsiteDataTypeServiceWorkerRegistrations not bound
 !missing-protocol-conformance! WebView should conform to NSUserInterfaceValidations (defined in 'WebIBActions' category)
```

Comment:
it this something fixed in master ?
or does it need something else in foundation (NS prefix) ?

Reply:
Completely unsure. It wasn't in the web headers so I assumed it'd come up in Foundation later. I can dig more now if needed, but assumed if it's in todo it's ok.